src/core/evaluator.js

```diff
@@ -978,7 +986,7 @@ class PartialEvaluator {
     });
   }
 
-  loadFont(fontName, font, resources) {
+  loadFont(fontName, font, resources, fallbackFontDict) {
```

Comment:
Please indicate that the parameter is optional here, i.e.
```suggestion
  loadFont(fontName, font, resources, fallbackFontDict = null) {
```